db/change_listener.go

```diff
@@ -278,15 +244,6 @@ func (listener *changeListener) NotifyCheckForTermination(ctx context.Context, k
 	listener.tapNotifier.L.Unlock()
 }
 
-func (listener *changeListener) notifyStopping(ctx context.Context) {
```

Comment:
This doesn't seem like it should be tap-specific.  Are we sure it's not an oversight that we don't handle this in the DCP case?

Reply:
I also wondered about this if there is a latent bug here (this code is unused). I think can affect things like the change feed not exiting because nothing goes to wake up the sync.Cond - so in changes.go `changeWaiter.Wait()` may not get woken up.

This code is not used today though.